resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml

```diff
@@ -40,15 +40,47 @@ spec:
                   type: object
                   x-kubernetes-preserve-unknown-fields: true
               metadata:
-                x-kubernetes-preserve-unknown-fields: true
+                default: {}
+                properties:
+                  collection:
+                    default: false
+                    type: boolean
+                  force:
+                    default: false
+                    type: boolean
+                  image:
+                    default: "quay.io/cloud-bulldozer/backpack:latest"
+                    type: string
+                  label:
+                    type: object
+                    x-kubernetes-preserve-unknown-fields: true
+                  privileged:
+                    default: false
+                    type: boolean
+                  serviceaccount:
+                    default: default
+                    type: string
+                  stockpileSkipTags:
+                    default: []
+                    items:
+                      type: string
+                    type: array
+                  stockpileTags:
+                    default: ["memory", "k8s", "openshift"]
```

Comment:
why not "common" in the default tags as well?

Reply:
Sure, had a bug here :). The documentation said `["common", "k8s", "openshift"]`.